USAGE.md

```diff
@@ -32,10 +32,9 @@ Git directories are searched for the latest commit hash. Searching for git commi
 
 ### Ignored files
 
-By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. If the specified file is not part of a git repository, `.gitignore` files are parsed recursively starting from the target directory,
-otherwise they are parsed from the root of the git repository as typical.
+By default, OSV-Scanner will not scan files that are ignored by `.gitignore` files. All recursively scanned files are matched to a git repository (if it exists) and any matching `.gitignore` files within that repository are taken into account.
```

Comment:
I don't know if something about #209 is worth mentioning here

Reply:
Let's just link it to be clear to people about known bugs.

Reply:
Added a link